gh-url.el

```diff
@@ -152,7 +152,7 @@
            (gh-url-run-request req resp)))))))
 
 (defun gh-url-form-encode (form)
-  (mapconcat (lambda (x) (format "%s=%s" (car x) (cdr x)))
+  (mapconcat (lambda (x) (format "%s=%s" (car x) (car (cdr x))))
```

Comment:
Nevermind this, I think I found an example of how to pass these args correctly. Maybe I'll add a docstring as well.